kayenta-signalfx/src/main/java/com/netflix/kayenta/signalfx/metrics/SignalFxMetricsService.java

```diff
@@ -144,8 +144,9 @@ public List<MetricSet> queryMetrics(
       signalFlowExecutionResult =
           signalFlowService.executeSignalFlowProgram(
               accessToken, startEpochMilli, endEpochMilli, stepMilli, maxDelay, immediate, program);
-    } catch (RetrofitError e) {
-      ErrorResponse errorResponse = (ErrorResponse) e.getBodyAs(ErrorResponse.class);
+    } catch (SpinnakerServerException e) {
```

Comment:
This is actually a change/reduction in functionality (olnly the message property instead of all ErrorResponse fields).  There's a plan for SpinnakerServerException (or perhaps it'll be only SpinnakerHttpException) to have an accessor for json response bodies as a Map, so perhaps it's better to wait for that?

Reply:
I'll revert this change and add a TODO with the note to replace RetrofitError once the Spinnaker exceptions have a response body accessor

Reply:
thanks, I'll check it out and make the necessary changes

Reply:
it looks like kayenta pulls in kork through its orca dependency unless I'm mistaken. I see that orca's main branch has the latest kork version with the accessor present but there hasn't been a release of orca with that change yet.

Reply:
Aaah, right you are.  Lemme tag orca so kayenta picks up the new stuff.

Reply:
And finally with https://github.com/spinnaker/kayenta/pull/974, kayenta gets the new stuff.